stable/lamp/templates/deployment.yaml

```diff
@@ -400,7 +400,7 @@ spec:
         {{- if .Values.php.repository }}
         image: "{{ .Values.php.repository }}:{{ .Values.php.tag }}"
         {{- else }}
-        image: "php:{{ .Values.php.version }}-fpm"
+        image: "php:{{ .Values.php.version }}-fpm-alpine"
```

Comment:
we should probably pull this image repo from a value so it can be pulled from a non public repo.